### PR TITLE
Completed javascript-modal. | Vera Yeh

### PR DESCRIPTION
**Pull request by VeraYeh**
image
image

### CODE REVIEW

File: javascript-modal/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <head>
+    <title>Javascript Modal</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+  </head>
+  <body>
+    <div>
+      <button class="open-button">Open Modal</button>
+    </div>
+    <div class="modal">
+      <fieldset>
```
Comment (BrettAlbright) [inline javascript-modal/index.html]: fieldset and label are used within a form element. I would remove these and use standard text elements if you are not using a form.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

File: javascript-modal/index.html (hunk 1 above)
Comment (VeraYeh) [reply javascript-modal/index.html]: Thanks for the comment. Updated html/css/js files accordingly. Please review, thanks.
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**